src/test/rgw/sfs/test_rgw_sfs_sqlite_buckets.cc

```diff
@@ -19,6 +19,39 @@ using namespace rgw::sal::sfs::sqlite;
 namespace fs = std::filesystem;
 const static std::string TEST_DIR = "rgw_sfs_tests";
 
+/*
```

Comment:
@jecluis @0xavi0 what do you think about this approach?
It is an hack to allow to fill, otherwise not fillable structs.
This is for testing only obviously.

Reply:
I must say I don't have an opinion on this. If you believe this to be the right approach to mock what needs to be mocked, I'm okay with it.